config/secrets.yml.sample

```diff
@@ -1,21 +1,29 @@
-slack_token: XXXXXXXXXX
+# Details for the below keys are in LastPass under Shared Folders > Labs (secure note)
+# Also see the note for more information on what each key does
```

Comment:
I can't seem to see this shared folder

Reply:
Yeah unfortunately it seems to have disappeared. Will create it again

Reply:
As per conversation in Slack, I've added you as an administrator (including others that were missing) to this folder, which as you discovered(!) you needed to then re-login to see the folder in your vault